aws-api/src/main/java/com/amplifyframework/api/aws/sigv4/AppSyncSigV4SignerInterceptor.java

```diff
@@ -25,9 +25,15 @@
 import com.amazonaws.auth.AWSCredentialsProvider;
 import com.amazonaws.http.HttpMethodName;
 import com.amazonaws.util.IOUtils;
+import com.amazonaws.util.StringUtils;
```

Comment:
This enters the project from a transitive dependency. IMO we should minimize our use of `com.amazonaws.*` overall, and try to stick to public APIs related to the per-category functionalities.

Since you're only using it for an `isBlank(...)` check, use our `Empty.check(...)` instead.